packages/wdio-applitools-service/src/@types/applitools-service.d.ts

```diff
@@ -0,0 +1 @@
+../../applitools-service.d.ts
```

Comment:
Why is this needed?

Reply:
We need to have package's typing accessible in `src` directory and outside of it. I guess it's a way to generate typings for non-TS files, but I'm not sure if it's required for rewritten files.